sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessage.java

```diff
@@ -57,6 +57,17 @@
 
     private Context context;
 
+    /**
```

Comment:
I don't see any test that uses this constructor and verifies contents.

Reply:
added